### PR TITLE
https://github.com/ensime/emacs-sbt-mode/issues/44

### PR DESCRIPTION
**Pull request by VyacheslavMik**

### CODE REVIEW

**Comment by VyacheslavMik** [comment]
Latest commit fixing this [issue](https://github.com/ensime/emacs-sbt-mode/issues/41).

**Comment by fommil** [comment]
doh! I missed this

File: sbt-mode.el [hunk 1]
```diff
@@ -176,6 +176,11 @@ buffer called *sbt*projectdir."
         (message "Starting sbt in buffer %s " buffer-name)
         ;;(erase-buffer)
 
+	;; In windows set SBT_OPTS like "-Djline.terminal=jline.UnsupportedTerminal"
+	;; https://github.com/ensime/emacs-sbt-mode/issues/44
+	(when (and (eql system-type 'windows-nt) (not (getenv "SBT_OPTS")))
+	  (setenv "SBT_OPTS" "-Djline.terminal=jline.UnsupportedTerminal"))
```
Comment (fommil) [inline sbt-mode.el]: this will override anything the user has seen, you need to add to an existing envvar.

Comment (VyacheslavMik) [inline sbt-mode.el]: Fix it.

**Pull request merged**